replica-builder/deployer/main.go

```diff
@@ -16,18 +17,29 @@ var (
 
 func main() {
 
-	// get application config yaml as input
+	// get deployer config yaml as input
 	if len(os.Args) != 2 {
 		log.Fatal("Expected 1 argument, got ", len(os.Args)-1)
 	}
 
-	// init ApplicationConfig object
-	applicationConfig := config.ReadApplicationConfig(os.Args[1])
-	pretty.Printf("Exporting application %# v", applicationConfig)
+	// Unmarshal deployer config and get cluster and application configs
+	ComponentConfig := config.ReadDeployerConfig(os.Args[1])
```

Comment:
why capital C? I don't think this is public variable, right?

Reply:
you are right, its a typo from copy/paste, its supposed to be small c, ill push a fix right now